optin/puller/workflows/payout.py

```diff
@@ -15,10 +20,40 @@
 # TODO_:
 # 2. implement send_funds
 
+XYM_MOSAIC_ID = 0x6BED913FA20223F8
 
-def send_funds(_):
-	# TODO_: implement this
-	return Hash256('')
+
+def get_private_key(filepath):
+	password = getpass.getpass(f'Enter password for {filepath}: ')
```

Comment:
i thought this was supposed to be headless script ?

Reply:
a) I assume we wanted to pass password somehow
b) alternative is via ENV, can't say I'm a fan of that, esp if you have some special chars...